custom-recipes/ocr-text-extraction-dataset/recipe.py

```diff
@@ -14,25 +21,42 @@
 input_filenames = input_folder.list_paths_in_partition()
 total_images = len(input_filenames)
 
-df = pd.DataFrame()
+rows = []
 
 for i, sample_file in enumerate(input_filenames):
-    if sample_file.split('.')[-1] != "jpg":
-        logger.info("OCR - Rejecting {} because it is not a JPG file.".format(sample_file))
+    prefix = sample_file.split('.')[0]
+    suffix = sample_file.split('.')[-1]
+
+    if suffix not in Constants.TYPES:
+        logger.info("OCR - Rejecting {} because it is not a {} file.".format(sample_file, '/'.join(Constants.TYPES)))
         logger.info("OCR - Rejected {}/{} images".format(i+1, total_images))
```

Comment:
Now images -> files ?
```suggestion
        logger.info("OCR - Rejected {}/{} files".format(i+1, total_files))
```